python/lsst/pipe/tasks/htmIndexEngine.py

```diff
@@ -133,6 +133,135 @@ def run(self, parsedCmd):
                 result = result,
             )
 
+class IngestIndexedReferenceTask(pipeBase.CmdLineTask):
+    ConfigClass = IngestIndexedReferenceConfig
+    RunnerClass = IngestReferenceRunner
+    _DefaultName = 'IngestIndexedReferenceTask'
+
+    _flags = ['photometric', 'resolved', 'variable']
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        parser = pipeBase.ArgumentParser(name=cls._DefaultName)
```

Comment:
Can you expand on this?  I didn't know about InputOnlyArgumentParser.  I'm not reading the inputs from a repository, but I am writing to it.  That seems like the right way to go, but let me know if I'm wrong.